docs/source/concepts/DynamicalStructureFactorFromAbInitio.rst

```diff
@@ -24,147 +24,125 @@ initial guess for the structure of interest.
-The :math:`1 \rightarrow 0` events (i.e. energy *to* the scattered neutron) would be infrequent at experimental conditions and are neglected.
-In order to reconstruct the full spectrum one has to also consider higher-order quantum events.
-For second-order quantum events one should not only
-consider transitions :math:`0 \rightarrow 2`, but also simultaneous transitions :math:`0 \rightarrow 1`, :math:`0 \rightarrow 1'` for different phonons.
-Within the harmonic approximation all second-order transitions form the following set: :math:`\lbrace \omega_1 +
-\omega_1, \omega_1 + \omega_2, \omega_1 + \omega_3, \ldots, \omega_p + \omega_p \rbrace`.
-The cardinality of this set is :math:`p^2`, where :math:`p` is a number of fundamentals.
-In practice one can reduce this number by taking into consideration a realistic energy window
-and neglecting those :math:`\omega_{ij}=\omega_i + \omega_j` for which :math:`S(Q, \omega_i)` or :math:`S(Q, \omega_j)` is negligible.
-Within the harmonic approximation each phonon is treated as independent harmonic quantum oscillator.  The formula for :math:`S(Q, \omega_{ik})` is as follows [#Mitchell]_:
+In DFT studies of solid materials, the simulation region is generally a finite unit cell with periodic boundary conditions.
+This models an infinite perfect crystal; in order to compare such calculations with powder experiments, orientational averaging should be considered.
+In Abins, the "almost-isotropic approximation" is applied to the spherical integration over reciprocal space, replacing the vector :math:`\mathbf{Q}` with a scalar :math:`Q` [#Howard1983]_:sup:`,` [#Howard1983b]_:
 
-:math:`S^j(Q, \omega_{ik}) = \frac{Q^4}{15  C}\left( \mathrm{Tr}B^j_{\omega_i}\mathrm{Tr}B^j_{\omega_k} + B^j_{\omega_i}:B^j_{\omega_k} + B^j_{\omega_k}:B^j_{\omega_i} \right) exp\left(-Q^2 \beta^j coth^2\left(\frac{\hbar \omega_{ik}}{2 k_B T} \right) \right)\sigma^j`
+:math:`S_i (Q,\omega_\nu) = \sigma_i \frac{Q^2 \mathrm{Tr}\mathbf{B}_{\nu,i}}{3} \exp\left(-Q^2 \alpha_{\nu,i} \right)`
 
 where
 
-:math:`\beta^j = A^j / 3`.
-
-:math:`C` is a constant: if :math:`i=j` then :math:`C=2`, otherwise :math:`C=1`.
-
-Similarly, one can define the contribution for the third quantum order events (:math:`0 \rightarrow 3`, simultaneous  :math:`0 \rightarrow 1`, :math:`0 \rightarrow 1'`, :math:`0 \rightarrow 1''` for different oscillators, etc.) [#Mitchell]_:
+:math:`\alpha_{\nu,i} = \frac{1}{5} \left \lbrace \mathrm{Tr} \mathbf{A}_i  + \frac{2 \mathbf{B}^{\omega_\nu,i}: \mathbf{A}_i}{\mathrm{Tr} \mathbf{B}^{\omega_\nu,i}} \right\rbrace`
 
-:math:`S^j(Q, \omega_{ikl}) = \frac{9Q^6}{543}\left( \mathrm{Tr}B^j_{\omega_i} \mathrm{Tr}B^j_{\omega_k} \mathrm{Tr}B^j_{\omega_l}  \right)  exp\left(-Q^2 \beta^j coth^2\left(\frac{\hbar \omega_{ikl}}{2 k_B T}\right) \right)\sigma^j`.
+Where :math:`\mathrm{Tr}` denotes the "trace" and :math:`:` the tensor "contraction" operation.
```

Comment:
I think we have used the "contraction" in the equations earlier on in the doc. Is it worth defining this there?

Reply:
Would it be sufficient to add "(i.e. the trace of the dot product)" here or should I spell out the Einstein summation?

(Looking into this a little more, I think the definition is actually $A_{ij} B_{ij}$ and not $A_{ij} B_{ji}$... but the latter is how it is done in the code and because these matrices are symmetric by construction the result is the same.)

Reply:
I think spelling it out is a good idea. I think we should do it on line 60

Reply:
Ok, made a few additions to clarify this stuff.